ui/admin/tests/acceptance/groups/members-test.js

```diff
@@ -62,16 +59,23 @@ module('Acceptance | groups | members', function (hooks) {
   test('visiting group members', async function (assert) {
     await visit(urls.members);
     await a11yAudit();
+
     assert.strictEqual(currentURL(), urls.members);
-    assert.strictEqual(findAll('tbody tr').length, membersCount);
+    assert.strictEqual(findAll(commonSelectors.TABLE_ROW).length, membersCount);
```

Comment:
nitpick: These type of selectors can be updated to `assert.dom(commonSelectors.TABLE_ROW).exists({ count: membersCount });`

Reply:
thank you! updated to use these